### PR TITLE
added info on how to start influx

### PR DESCRIPTION
**Pull request by lwandzura**
Closes https://github.com/influxdata/docs-v2/issues/4053

added that influxd must be running in order to work with influx
added instruction on how to start influx

### CODE REVIEW

File: content/influxdb/v2.1/get-started.md [hunk 1]
```diff
@@ -19,6 +19,16 @@ After you've [installed InfluxDB OSS](/influxdb/v2.1/install/), you're ready to
 - [Visualize data](#visualize-data)
 - [Monitor and alert](#monitor-and-alert)
 
+**NOTE:** You **must** start the influxd daemon in order to start working with InfluxDB.  See [InfluxDB service](/influxdb/v2.1/reference/cli/influxd/). For information on the command line interface (CLI) see [InfluxDB command line interface](/influxdb/v2.1/reference/cli/influx/).  Once you've started the influxd daemon, use `localhost:8086` to log in to your instance. 
+
+To start Influx do the following:
+  1. Open a terminal.
+  2. Type influxd in the command line.
```
Comment (kelseiv) [inline content/influxdb/v2.1/get-started.md:26]:
```suggestion
To start InfluxDB, do the following:
  1. Open a terminal.
  2. Type `influxd` in the command line.
```

File: content/influxdb/v2.2/get-started.md [hunk 2]
```diff
@@ -19,6 +19,16 @@ After you've [installed InfluxDB OSS](/influxdb/v2.2/install/), you're ready to
 - [Visualize data](#visualize-data)
 - [Monitor and alert](#monitor-and-alert)
 
+**NOTE:** You must start the influxd daemon in order to start working with InfluxDB.  See [InfluxDB service](/influxdb/v2.2/reference/cli/influxd/). For information on the command line interface (CLI) see [InfluxDB command line interface](/influxdb/v2.2/reference/cli/influx/).  Once you've started the influxd daemon, use `localhost:8086` to log in to your instance. 
+
+To start Influx do the following:
+  1. Open a terminal.
+  2. Type influxd in the command line.
```
Comment (kelseiv) [inline content/influxdb/v2.2/get-started.md:26]:
```suggestion
To start InfluxDB, do the following:
  1. Open a terminal.
  2. Type `influxd` in the command line.
```

File: content/influxdb/v2.0/get-started.md [hunk 3]
```diff
@@ -19,6 +19,16 @@ After you've [installed InfluxDB OSS](/influxdb/v2.0/install/), you're ready to
 - [Visualize data](#visualize-data)
 - [Monitor and alert](#monitor-and-alert)
 
+**NOTE:** You must start the influxd daemon in order to start working with InfluxDB.  See [InfluxDB service](/influxdb/v2.0/reference/cli/influxd/). For information on the command line interface (CLI) see [InfluxDB command line interface](/influxdb/v2.0/reference/cli/influx/).  Once you've started the influxd daemon, use `localhost:8086` to log in to your instance.
```
Comment (kelseiv) [inline content/influxdb/v2.0/get-started.md:22]:
```suggestion
**Note:** To run InfluxDB, start the `influxd` daemon ([InfluxDB service](/influxdb/v2.0/reference/cli/influxd/)) using the [InfluxDB command line interface](/influxdb/v2.0/reference/cli/influx/). Once you've started the `influxd` daemon, use `localhost:8086` to log in to your instance.
```

File: content/influxdb/v2.1/get-started.md [hunk 4]
```diff
@@ -19,6 +19,16 @@ After you've [installed InfluxDB OSS](/influxdb/v2.1/install/), you're ready to
 - [Visualize data](#visualize-data)
 - [Monitor and alert](#monitor-and-alert)
 
+**NOTE:** You **must** start the influxd daemon in order to start working with InfluxDB.  See [InfluxDB service](/influxdb/v2.1/reference/cli/influxd/). For information on the command line interface (CLI) see [InfluxDB command line interface](/influxdb/v2.1/reference/cli/influx/).  Once you've started the influxd daemon, use `localhost:8086` to log in to your instance.
```
Comment (kelseiv) [inline content/influxdb/v2.1/get-started.md:22]:
```suggestion
*Note:** To run InfluxDB, start the `influxd` daemon ([InfluxDB service](/influxdb/v2.1/reference/cli/influxd/)) using the [InfluxDB command line interface](/influxdb/v2.1/reference/cli/influx/). Once you've started the `influxd` daemon, use `localhost:8086` to log in to your instance.
```

File: content/influxdb/v2.2/get-started.md [hunk 5]
```diff
@@ -19,6 +19,16 @@ After you've [installed InfluxDB OSS](/influxdb/v2.2/install/), you're ready to
 - [Visualize data](#visualize-data)
 - [Monitor and alert](#monitor-and-alert)
 
+**NOTE:** You must start the influxd daemon in order to start working with InfluxDB.  See [InfluxDB service](/influxdb/v2.2/reference/cli/influxd/). For information on the command line interface (CLI) see [InfluxDB command line interface](/influxdb/v2.2/reference/cli/influx/).  Once you've started the influxd daemon, use `localhost:8086` to log in to your instance.
```
Comment (kelseiv) [inline content/influxdb/v2.2/get-started.md:22]:
```suggestion
*Note:** To run InfluxDB, start the `influxd` daemon ([InfluxDB service](/influxdb/v2.2/reference/cli/influxd/)) using the [InfluxDB command line interface](/influxdb/v2.2/reference/cli/influx/). Once you've started the `influxd` daemon, use `localhost:8086` to log in to your instance.
```

File: content/influxdb/v2.1/get-started.md [hunk 6]
```diff
@@ -19,6 +19,16 @@ After you've [installed InfluxDB OSS](/influxdb/v2.1/install/), you're ready to
...
 - [Monitor and alert](#monitor-and-alert)
 
+**NOTE:** You **must** start the influxd daemon in order to start working with InfluxDB.  See [InfluxDB service](/influxdb/v2.1/reference/cli/influxd/). For information on the command line interface (CLI) see [InfluxDB command line interface](/influxdb/v2.1/reference/cli/influx/).  Once you've started the influxd daemon, use `localhost:8086` to log in to your instance. 
+
+To start Influx do the following:
+  1. Open a terminal.
+  2. Type influxd in the command line.
+
+```bash
+influxd
```
Comment (kelseiv) [inline content/influxdb/v2.1/get-started.md:29]: to be consistent in our code examples, we typically use `sh` in lieu of `bash`
(unless there's a technical distinction to be made (eg, you must use `bash` bc `sh` doesn't support that functionality).

**Comment by kelseiv** [comment]
Thanks! Approve w minor edits.

File: content/influxdb/v2.1/get-started.md (hunk 6 above)
Comment (lwandzura) [reply content/influxdb/v2.1/get-started.md:29]: thanks Kelly

**Pull request merged**